CONTRIBUTING.md

```diff
@@ -115,6 +115,12 @@ more may be necessary depending on the case. Here is an example used to uplift f
    * `./examples/electron/package.json`
    * `./package.json`
    * `./theia-extensions/viewer-prototype/package.json`
+1. Align these React dependency versions in `./package.json` with Theia's `packages/core/package.json`, when applicable:
```

Comment:
Same here, can you continue the numbering?

Reply:
-Per above. Thanks for double-checking from your end.